clippy_lints/src/doc.rs

```diff
@@ -413,6 +416,12 @@ fn check_doc<'a, Events: Iterator<Item = (pulldown_cmark::Event<'a>, Range<usize
                 };
                 let (begin, span) = spans[index];
                 if in_code {
+                    let lo = span.lo() + BytePos::from_usize(range.start - begin);
+                    let span = Span::new(
+                        lo,
+                        lo + BytePos::from_usize(text.len() + count(text.as_bytes(), b'\n') * 4),
```

Comment:
Can you add a comment, what this `BytePos` is. Preferably with a small example.

Reply:
Done.